### ISSUE TITLE
Move and rename `sdg-elements` directory and refactor associated code

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4183]
### Overview
We want to move the `sdg-elements` folder up a level from the `assets/images/about` to the `assets/images` directory so that it accurately reflects that we use those images on multiple pages on our website.

### Details
SDG is an acronym for the [United Nations Development Programme Sustainable Development Goals](https://www.undp.org/sustainable-development-goals).

### Action Items
- [x] Move the file directory `assets/images/about/sdg-elements` up one level so it becomes `assets/images/sdg-elements`
- [x] Rename the directory `sdg-elements` to `sdg`
- [x] As a result of the above changes, change the paths of the SDG images in the following files:
  - [x] _includes/about-page/about-card-sustainability.html
  - [x] _projects/311-data.md
  - [x] _projects/access-the-data.md
  - [x] _projects/engage.md
  - [x] _projects/lucky-parking.md
  - [x] _projects/open-community-survey.md
- [x] Using the search function in VS Code or IDE, search for `assets/images/about/sdg-elements`. If you find any files with that, update the code in those files to the new path.
- [x] Check the following pages look the same as the pages on the live website
  - [x] About page: https://www.hackforla.org/about/#sustainability
  - [x] Citizen Engagement page: https://www.hackforla.org/citizen-engagement
  - [x] 311 Data: https://www.hackforla.org/projects/311-data
  - [x] Access the Data: https://www.hackforla.org/projects/access-the-data
  - [x] Engage: https://www.hackforla.org/projects/engage
  - [x] Lucky Parking: https://www.hackforla.org/projects/lucky-parking
  - [x] Open Community Survey: https://www.hackforla.org/projects/open-community-survey

#### Merge Team, Technical Lead, or Product Manager
- [x] Once the pull request associated with this issue is merged, go to each of the following issues and check if all the issues listed under the `Dependency` section are closed. If the dependencies are all closed, move those issues into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it.
  - [x] #4254
  - [x] #4255
  - [x] #4256
  - [x] #4225
  - [x] #4257
  - [x] #4253
  - [x] #4258  

#### Files that need the paths of the SDG images to be updated
- [_includes/about-page/about-card-sustainability.html](https://github.com/hackforla/website/blob/gh-pages/_includes/about-page/about-card-sustainability.html)
- [_projects/311-data.md](https://github.com/hackforla/website/blob/gh-pages/_projects/311-data.md?plain=1)
- [_projects/access-the-data.md](https://github.com/hackforla/website/blob/7770fdeaa68f44c6ceb32866a24b903ad52c2019/_projects/access-the-data.md?plain=1)
- [_projects/engage.md](https://github.com/hackforla/website/blob/gh-pages/_projects/engage.md?plain=1)
- [_projects/lucky-parking.md](https://github.com/hackforla/website/blob/gh-pages/_projects/lucky-parking.md?plain=1)
- [_projects/open-community-survey.md](https://github.com/hackforla/website/blob/gh-pages/_projects/open-community-survey.md?plain=1)

#### Pages to check to make sure pages stay the same
- About page: https://www.hackforla.org/about/#sustainability
- Citizen Engagement page: https://www.hackforla.org/citizen-engagement
- 311 Data: https://www.hackforla.org/projects/311-data
- Access the Data: https://www.hackforla.org/projects/access-the-data
- Engage: https://www.hackforla.org/projects/engage
- Lucky Parking: https://www.hackforla.org/projects/lucky-parking
- Open Community Survey: https://www.hackforla.org/projects/open-community-survey
- This issue is tracked in #4153

**Comment by JessicaLucindaCheng** [comment]
Question for Product Manager:
![sdg-la](https://user-images.githubusercontent.com/31293603/225156215-e5fe08dd-9ba9-4eb2-914c-b9bfab33ab3c.svg)
I'm trying to figure out if this image should be in the same directory as the other SDG images. Will this image be used anywhere else on the site?

**Comment by JessicaLucindaCheng** [comment]
Once this issue is in the New Issue Approval column, a dev lead needs to review the issue and determine if it is ready for a PM to prioritize it. If it isn't ready, a dev lead will need to edit this issue so that it is ready for a PM to prioritize it.

**Comment by arcan9** [comment]
Availability: 5-8pm, M-F
ETA: EOD Fri

**Comment by ExperimentsInHonesty** [comment]
@arcan9 congrats for taking on this issue.  Please remember in future to move it into the in progress after you self assign.
